File: C programing/assignment3/perfect.c
// write a code to check is number perfect or not

#include<stdio.h>
void main(){
    int n;
    printf("enter the number");
    scanf("%d",&n);
    int sum=0;
    int i=1;
    while(i<=n/2){
        if(n%i==0){
            sum=sum+i;
        }
        i++;
    }
    if(n==sum){
        printf("is perfect numner");
    }
    else{
        printf("is not perfect number");
    }
}

File: core java/src/Perfect.java

class Perfect {

	public static void main(String[] args) {
		 int sum=333;
		   System.out.println("enter the number");
		    int sum1=0;
		    int i=1;
		    while(i<=sum1/2){
		        if(sum1%i==0){
		            sum1=sum1+i;
		        }
		        i++;
		    }
		    if(sum1==sum1){
		        System.out.println("is perfect numner");
		    }
		    else{
		    	System.out.println("is not perfect number");
		    }

	}

}
